packages/react-google-maps-api/src/components/places/StandaloneSearchBox.tsx

```diff
@@ -1,38 +1,45 @@
 /* global google */
-import React, {
-  PureComponent,
-  Children
-} from 'react'
+import * as React from 'react'
```

Comment:
IMHO for me explicit imports tells a lot about file functionality. I do not like `import *` it is better to use

```
import {
  default as React,
  PureComponent,
  Children
} from 'react'

Reply:
Fixed (and in 3 other components with same issue)